mailing_list/templates/mailing_list/index.html

```diff
@@ -44,5 +21,76 @@
             RESOURCE_LINK_ID: '{{ resource_link_id }}'
         };
     </script>
+    <title>Mailing List</title>
+
+</head>
+<body class="lti-tool" role="application" ng-controller="MailingListController as ml">
+    <header>
+        <h1>{% block breadcrumbs %}Course Emailer{% endblock breadcrumbs %}</h1>
+    </header>
+    <div class="loading-indicator" ng-show="ml.isLoading">
+        <img src="{% static 'mailing_list/images/ajax-loader-large.gif' %}"/>
+    </div>
+    <main class="ng-hide" ng-show="ml.loaded">
+        <div id="course-sections" ng-show="ml.hasPrimarySections()">
+            <h2>Course Sections</h2>
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th>Section Name</th>
+                        <th>Mailing List Address</th>
+                        <th>Number of Recipients</th>
+                        <th>Who can post to this list?</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr ng-repeat="list in ml.primarySectionLists">
+                        {% verbatim %}
+                        <td>{{ list.name }}</td>
+                        <td>
+                            <span ng-show="list.access_level == 'readonly'">{{ list.address }}</span>
+                            <a href="mailto:{{ list.address }}" ng-show="list.access_level != 'readonly'">{{ list.address }}</a>
+                        </td>
+                        <td>{{ list.members_count }}</td>
+                        {% endverbatim %}
+                        <td>
+                            <select ng-model="list.access_level" ng-options="l.id as l.name for l in ml.accessLevels" ng-change="ml.updateAccessLevel(list)"></select>
+                            <img src="{% static 'mailing_list/images/ajax-loader-small.gif' %}" ng-show="list.isUpdating"/>
+                            <span class="text-success feedback" ng-show="list.updated">&#10003;</span>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+        <div id="other-sections" ng-show="ml.hasOtherSections()">
+            <h2>Other Sections</h2>
+            <table class="table">
```

Comment:
suggestion for next time: put the table generation in a separate template, then [`include`](https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#include) it for each set of lists